Formula/go.rb

```diff
@@ -3,6 +3,10 @@ class Go < Formula
   homepage "https://golang.org"
 
   stable do
+    # Version bumps of this formula require formulae that depend on Go at
```

Comment:
#6096, #7452, and #9304 would seem to indicate it's only major/minor and not patch versions